.pre-commit-config.yaml

```diff
@@ -21,3 +21,12 @@ repos:
         name: Check for NO_COMMIT marker
         entry: bash -c "git diff --cached -U0 | (! grep NO_COMMIT)"
         language: system
+        require_serial: true
+
+  - repo: local
+    hooks:
+      - id: mypy
+        name: mypy
+        entry: mypy --no-error-summary
```

Comment:
Just a bit spamming because it splits the `mypy` run to multiple processes and then outputs a line even when it succeeds, e.g:
<img width="314" alt="image" src="https://github.com/elementary-data/elementary/assets/11314965/8f8b264e-5c84-4a38-abde-9446f1fe1e7b">

If there are errors they will still be printed